README.md

```diff
@@ -1,3 +1,18 @@
 # joy-generator
 
 ArgoCD plugin for expanding joy releases for the ArgoCD ApplicationSet Controller.
+
+## How-to roll-out and test updates
+
+1. Update this repo and merge to master (triggering a build & /release)
+    1. Note the new release version# from https://github.com/nestoca/joy-generator/releases
+    2. Ensure the same version is present in 
+2. Apply the newly released version in `infra`'s component `ArgoCD.configureJoyGenerator()`'s `devVersion` [here](https://github.com/nestoca/infra/blob/c25905374ec678ec21ce468cb00628b572f70838/pulumi/platform/components/ArgoCD.ts#L478)
+    1. do `pu main preview-markdown` on `infra/pulumi/platform` as part of your PR/roll-out & merge, as normal
+4. Test the new version
+    1. ℹ️ `devVersion` will be deployed to environments that contain `dev` in their name, `version` to all other envs
```

Comment:
```suggestion
    1. ℹ️ `devVersion` will be deployed to `dev` environment, `version` to all other envs
```

Reply:
That's true today; I was aligning on the logic so that if someone creates an environment called `medevac` (😓), they can expect to get the `devVersion` of joy-generator.

Reply:
You can link to the section if you want, just for reference. However, as I understand it, the selector matches the exact string `dev`, not substrings. Not super clear in [docs](https://argo-cd.readthedocs.io/en/stable/operator-manual/applicationset/Generators-Post-Selector/), but quite sure! 😉

Reply:
The default settings for [applying?] `joy-generator-config` is (`dev NotIn environment.metadata.name`): https://github.com/nestoca/infra/blob/e4a7fe8fd3aa11c5cba44f3e2a94258b7e208afc/pulumi/platform/components/argocd/catalog-env-releases.yaml#L28-L33

Which is copied, then flipped to `In` for `joy-generator-config-dev`: https://github.com/nestoca/infra/blob/e4a7fe8fd3aa11c5cba44f3e2a94258b7e208afc/pulumi/platform/components/ArgoCD.ts#L605

I guess the link I'm missing is from these `metadata.name` values to the env/project name(s)?